geps/gep-1709.md

```diff
@@ -538,6 +556,14 @@ iteration to add this if desired, but it probably warrants its own GEP as we
 need to make sure we have buy-in from multiple stakeholders with different
 implementations that are implementing those features.
 
+### Mesh Profiles
+
+We eventually want profiles for GAMMA/mesh related tests, however at this point
+we believe the test suite for mesh could end up being it's own separate thing
```

Comment:
It might fit still, we're just calling this out as an alternatives considered because we don't expect we need to solve this right now (in fact we can't).